RFC-by-stage/1-approved/rename-system.md

```diff
@@ -0,0 +1,37 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Rename design system to GOLD
+
+I propose that we rename "Australian Government Design System" to "GOLD Design System", where GOLD is an acronym for "Government Open Language for Design". I also propose we rename "Design System AU" to "Design System Au", where "u" in "Au" is lower case as that the atomic label representing the element gold. I think this is makes the connection between the org naming and DS naming clearer and more logical.
+
+I propose that we do this as it was the original name of the system, as can be seen in this [CSS Tricks tweet](https://mobile.twitter.com/css/status/970805659842134016) and the [original docs site](https://trevorbrennan.com/cv/gold).
+
+I am aware this is supported by @dominikwilkowski for different reasons, mentioned [in his Twitter thread](https://twitter.com/wilkowskidom/status/1426350990446399490?s=20). Dominik argues that having a fun name (as opposed to a plain language name) has the effect of "creating shared language" and is better to "excite people about working together", particularly non-technical people. That this is beneficial for "creating unity and delight" and gives your people an identity. Further, that not taking a project in a light-hearted way ("not too seriously") is "an important part to make yourself #accessible".
+
+Building on this, in light of the end of an era of DTA custodianship, I believe this would be an appropriate identity change to reignite the community.
+
+There is a counter argument for leaving the name as "Australian Government Design System" or "AuDS" because it is more clear, and people know what it is.
+
+However, both are authentic names for the system.
+
+Also, and finally, everyone loved the name "GOLD".
+
+## A short description of the desired outcome or solution
+
+This would require:
+
+- to update all references of "Australian Government Design System" to "GOLD Design System"
+- to update of all references of "AuDS" to "GOLD"
+- Other branding updates
+- A Press Release or Blog post indicating this has occurred
+
+This would **not** require:
+
+- Change to the npm handle or Github org, as the organisation would be the same
```

Comment:
So to be clear, will we be on the @designsystemau scope? Will we need a one pager on migration steps?

Reply:
We should make it clear how to migrate yes. But it should be as simple as this inside the `package.json`:
```diff
 "dependencies": {
-    "@gov.au/accordion": "^3.0.0",
-    "@gov.au/animate": "^3.0.0",
-    "@gov.au/body": "^3.0.0",
-    "@gov.au/breadcrumbs": "^3.0.0",
-    "@gov.au/buttons": "^3.0.0",
+    "@designsystemau/accordion": "^3.0.0",
+    "@designsystemau/animate": "^3.0.0",
+    "@designsystemau/body": "^3.0.0",
+    "@designsystemau/breadcrumbs": "^3.0.0",
+    "@designsystemau/buttons": "^3.0.0",
}
```

Reply:
We have the scope `@designsystem.au`. I just pinged #feature-contin.. channel, do we potentially want to use another scope `@gold` for instance, instead?